hardware/Huawei_Atlas800TA3/FlagScale/run.sh.patch

```diff
@@ -1,6 +1,6 @@
 diff --git a/run.sh b/run.sh
 new file mode 100644
-index 000000000..072a78c94
+index 00000000..072a78c9
 --- /dev/null
 +++ b/run.sh
```

Comment:
Do we really need this? If necessary, it will be better to move it to examples/qwen2_5_vl/conf

Reply:
> Do we really need this? If necessary, it will be better to move it to examples/qwen2_5_vl/conf

Just a sample. I'll delete it afterwards.